docs/reference/transform/setup.asciidoc

```diff
@@ -5,39 +5,95 @@
 <titleabbrev>Setup</titleabbrev>
 ++++
 
-To use the {transforms}, you must have the
-{subscriptions}[appropriate license] and at least one
-<<transform-setup-nodes,{transform} node>> in your {es} cluster. If {stack}
-{security-features} are enabled, you must also ensure your users have the
-<<transform-privileges,necessary privileges>>.
-
 [discrete]
-[[transform-setup-nodes]]
-== {transform-cap} nodes
+[[requirements-overview]]
+== Requirements overview
+
+To use {transforms}, you must have:
 
-To use {transforms}, there must be at least one {transform} node in your cluster.
-If you want to control which nodes run {transforms}, add or remove `transform`
-from the `node.roles` setting on some nodes. For more information, see
-<<modules-node>> and <<transform-settings>>.
+[%interactive]
+- [ ] the {subscriptions}[appropriate license]
+- [ ] at least one <<transform-node,{transform} node>>
+- [ ] security privileges assigned to users that:
```

Comment:
nit: although it's obvious from the context that the verb "grant" in line 20 refers to noun "privileges", at first I read it as if it referred to the noun "users".

Feel free to leave it as is provided this is typical practice in English.

Reply:
Good point, i've simplified this phrase in 63140b77aa69eb69981e9d32b7f26eb0c738b419